<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvPage</class>
 <widget class="QWizardPage" name="AdvPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 8pt;</string>
  </property>
  <property name="title">
   <string>Advanced Settings</string>
  </property>
  <property name="subTitle">
   <string>Please specify the advanced settings.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Settings for Windows</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>MSSFIX:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="cbMsfix">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Route method:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cmbRouteMethod">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Verbosity:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="txtVerbose">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Route delay:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="txtRouteDelay">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Mute:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="txtMute">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Float:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="cbNobind">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>No bind:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="cbFloat">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Comp-LZO:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="cbCompLzo">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Auth-User:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="cbAuthUserPass">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
